forms/consts.py

```diff
@@ -0,0 +1,7 @@
+import settings
+import re
+import hashlib
+
+HASH = lambda x, y: hashlib.md5(x+y+settings.NONCE_SECRET).hexdigest()
+IS_VALID_EMAIL = lambda x: re.match(r"[^@]+@[^@]+\.[^@]+", x)
+EXCLUDE_KEYS = ['_gotcha', '_next', '_subject', '_cc']
```

Comment:
Yes, I agree. I would also suggest making IS_VALID_EMAIL and HASH named functions within helpers .py instead of "constants".